src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! Library Template for Rust
+//! This is a crate description, needed or else missing_docs warning will occur
```

Comment:
Just something on my mind regarding styling: do we want to end the sentence without a period? 

I think we should enforce appending periods to sentences because I can see us eventually generate docs from these comments, and serve them to our website. What do you think?

Reply:
Added periods: https://github.com/Arrow-air/template-rust-lib/pull/1/commits/10f99ba487d694fce868138e25b71987138371f9